common/datasets/librispeech/cart.py

```diff
@@ -0,0 +1,939 @@
+                                "values": "AA",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_AE",
+                                "values": "AE",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_AH",
+                                "values": "AH",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_AO",
+                                "values": "AO",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_IY",
+                                "values": "IY",
+                            },
+                            {
+                                "type": "question",
+                                "description": "sz",
+                                "values": "S Z",
+                            },
+                            {
+                                "type": "question",
+                                "description": "kg",
+                                "values": "K G",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_EH",
+                                "values": "EH",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_OY",
+                                "values": "OY",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_OW",
+                                "values": "OW",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_IH",
+                                "values": "IH",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_EY",
+                                "values": "EY",
+                            },
+                            {
+                                "type": "question",
+                                "description": "vowel_ER",
+                                "values": "ER",
+                            },
+                        ],
+                    },
+                ],
+            },
+        ]
+
+        if not add_unknown:
+            self.steps[1]["questions"].pop(1)
```

Comment:
Ah I see. This answers one of my comments above.

But in case anybody modifies the order of questions in the future I am afraid that we then unintentionally remove the wrong one.
I would rather that we add the question if add_unknown or at least add an assert that the name is as we expect it.

when I check "request changes" later this is what I refer to.